src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/PersonAbsenceTests.cs

```diff
@@ -211,6 +211,27 @@ public async Task CreateAbsence_ShouldFail_WhenSettingTaskDetailsOnType(ApiPerso
             response.Should().BeBadRequest();
         }
 
```

Comment:
Could you make a test that fails for both create & update ?
Given task details, but no base position or rolename.

Not only the "happy path"